projects/canopy/src/lib/pipes/pipes.module.ts

```diff
@@ -11,4 +11,14 @@ const pipes = [LgCamelCasePipe, LgKebabCasePipe];
   declarations: [...pipes],
   exports: [...pipes],
 })
-export class LgPipesModule {}
+/**
+ * @deprecated Pipe module is deprecated.
+ * Please import camel-case or kebab-case module instead.
+ */
+export class LgPipesModule {
+  constructor() {
+    console.warn(
+      'Pipe module is deprecated. Please import camel-case or kebab-case modules instead',
```

Comment:
I think it would be great to include the names of the module classes, like this:

LgPipesModule module is deprecated. Please import LgCamelCasePipeModule or LgKebabCasePipeModule instead.

Reply:
Absolutely